### PR TITLE
Add initial troubleshooting doc

### PR DESCRIPTION
**Pull request by douglas-reid**
Taken from content on team drive

### CODE REVIEW

File: troubleshooting/index.md [hunk 1]
```diff
@@ -10,3 +10,124 @@ type: markdown
...
+        proxy_http_version 1.1;
+        proxy_set_header Connection "";
+        ...
+    }
+}
+```
+
+## Running multiple replicas of Mixer does not work as expected 
+
+For the alpha release, Mixer **must** be configured to run as a single instance within a cluster. We are working on improvements at the protocol, configuration, and deployment levels to support multiple instance and high-availability deployments. We expect to remove this limitation shortly after the initial alpha release.
```
Comment (geeknoid) [inline troubleshooting/index.md]: Use the version identifier instead of Alpha through the doc. It's {{stie.data.istio.version}}

Comment (geeknoid) [inline troubleshooting/index.md]: multiple instance -> multiple instances

File: troubleshooting/index.md [hunk 2]
```diff
@@ -10,3 +10,124 @@ type: markdown
...
+## The Mixer pod was restarted and I lost my configuration updates
+
+For the alpha release, Mixer configuration is stored in a local file system-based store. By default, Mixer is not configured to use a kubernetes persistent volume.
+
+There are a few possible workarounds:
+
+1.  Save configuration updates to a common location and script the application of these updates via istioctl.
+1.  Configure a persistent volume and update the Mixer deployment specs to use that volume for the file system store.
+
+Work is ongoing to provide a highly-available, persistent configuration store for Mixer config. We expect this to work to land immediately following the initial alpha release.
```
Comment (geeknoid) [inline troubleshooting/index.md]: for Mixer config -> for Mixer

File: troubleshooting/index.md [hunk 3]
```diff
@@ -10,3 +10,124 @@ type: markdown
...
+
+### Configuring a persistent volume for Mixer config
+
+A Mixer deployment can be modified to use [kubernetes ConfigMaps](https://kubernetes.io/docs/tasks/configure-pod-container/configmap/) to provide persistent access to Mixer config. Kubernetes ConfigMaps are preserved across restarts and allow running multiple replicas of the mixer.
```
Comment (geeknoid) [inline troubleshooting/index.md]: ConfigMap should be in back ticks

Comment (geeknoid) [inline troubleshooting/index.md]: of the mixer -> of Mixer

File: troubleshooting/index.md [hunk 4]
```diff
@@ -10,3 +10,124 @@ type: markdown
...
+
+Note: `istioctl` is not useable for runtime updates to Mixer configuration when this approach is used (the updates will not persist).
```
Comment (geeknoid) [inline troubleshooting/index.md]: usable

File: troubleshooting/index.md [hunk 5]
```diff
@@ -10,3 +10,124 @@ type: markdown
...
+
+### With [Tcpdump](http://www.tcpdump.org/tcpdump_man.html)
+
+Tcpdump doesn't work in the sidecar pod - the container doesn't run as root. However any other container in the same pod will see all the packets, since the network namespace is shared. `iptables` will also see the pod-wide config. 
+
+Communication between Envoy and the app happens on 127.0.0.1, and is not encrypted. 
+
+## Envoy is crashing under load
+
+Check your `ulimit -a`. Many system have a 1024 open file descriptor limit by default which will cause envoy to assert and crash with:
```
Comment (geeknoid) [inline troubleshooting/index.md]: system -> systems

Comment (geeknoid) [inline troubleshooting/index.md]: envoy -> Envoy

File: troubleshooting/index.md (hunk 1 above)
Comment (ldemailly) [inline troubleshooting/index.md]: ^stie^site  (which is 0.1)

Comment (douglas-reid) [inline troubleshooting/index.md]: done.

File: troubleshooting/index.md (hunk 2 above)
Comment (douglas-reid) [inline troubleshooting/index.md]: done.

File: troubleshooting/index.md (hunk 3 above)
Comment (douglas-reid) [inline troubleshooting/index.md]: done.

Comment (douglas-reid) [inline troubleshooting/index.md]: done.

File: troubleshooting/index.md (hunk 4 above)
Comment (douglas-reid) [inline troubleshooting/index.md]: done.

File: troubleshooting/index.md (hunk 5 above)
Comment (douglas-reid) [inline troubleshooting/index.md]: done.

File: troubleshooting/index.md [hunk 6]
```diff
@@ -10,3 +10,124 @@ type: markdown
...
+
+## Running multiple replicas of Mixer does not work as expected 
+
+For the {{site.data.istio.version}} release, Mixer **must** be configured to run as a single instance within a cluster. We are working on improvements at the protocol, configuration, and deployment levels to support multiple instance and high-availability deployments. We expect to remove this limitation shortly after the initial {{site.data.istio.version}} release.
+
+Running multiple replicas of Mixer will lead to issues with configuration updates not propagating properly and improperly-enforced quotas (for the memQuota adapter).
+
+## The Mixer pod was restarted and I lost my configuration updates
+
+For the {{site.data.istio.version}} release, Mixer configuration is stored in a local file system-based store. By default, Mixer is not configured to use a kubernetes persistent volume.
```
Comment (geeknoid) [inline troubleshooting/index.md]: kubernetes -> Kubernetes, here and elsewhere

**Comment by douglas-reid** [comment]
PTAL. I've updated the docs to match the review comments.

File: troubleshooting/index.md (hunk 6 above)
Comment (douglas-reid) [inline troubleshooting/index.md]: Done.

**Pull request merged**